cypress/plugins/docker/index.ts

```diff
@@ -25,28 +26,32 @@ import PluginConfigOptions = Cypress.PluginConfigOptions;
 
 // A cypress plugin to run docker commands
 
-export function dockerRun(args: {
+export function dockerRun(opts: {
     image: string;
     containerName: string;
     params?: string[];
+    cmd?: string;
 }): Promise<string> {
     const userInfo = os.userInfo();
-    const params = args.params ?? [];
+    const params = opts.params ?? [];
 
-    if (userInfo.uid >= 0) {
+    if (params?.includes("-v") && userInfo.uid >= 0) {
         // On *nix we run the docker container as our uid:gid otherwise cleaning it up its media_store can be difficult
         params.push("-u", `${userInfo.uid}:${userInfo.gid}`);
     }
 
+    const args = [
+        "run",
+        "--name", `${opts.containerName}-${crypto.randomBytes(4).toString("hex")}`,
+        "-d",
+        ...params,
+        opts.image,
+    ];
+
+    if (opts.cmd) args.push(opts.cmd);
```

Comment:
If it is named `cmd` I would expect, that I can pass the command.
Maybe this should be renamed to `extraArgs`?

Reply:
You can? This is all valid Docker lingo, which confusingly puts the `CMD` _after_ the image name e.g it's of the form:
```
docker $action $flags $image $cmd
```
e.g:
```
docker run --rm -it postgres:13 /bin/sh
```